datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -292,7 +273,7 @@ def __get_versions(self, standard_distribution_name):
         try:
             # NOTE: We do not perform in-toto inspection for simple indices; only for wheels.
             index_abspath, _ = self.__download_with_tuf(index_relpath)
-        except UnknownTargetError:
+        except DownloadError:
```

Comment:
I think this is incorrect. I think what you want to check is that if `get_targetinfo()` returns `None`, then you want to return a `NoSuchDatadogPackage`.

Reply:
I see your point, `get_one_valid_targetinfo` used to raise `UnknownTargetError`, but `get_targetinfo()` returns `None` instead. I was trying to keep the changes to a minimum but this looks incorrect indeed, so needs a few more changes.

Reply:
Should be fixed now, that was a really good catch!

Reply:
Thanks! Let me take a look when I get back from sick leave next week, please...